src/clj/athens/self_hosted/event_log.clj

```diff
@@ -41,7 +57,9 @@
 (defn deserialize
   [{id   "event/id"
     data "event/data"}]
-  [id (edn/read-string data)])
+  [(if (str/blank? id)
```

Comment:
This really should not be done. We shouldn't make up a new `event/id` out of thin air. If an event does not have an ID, it is not valid.

What's the case where this was necessary? Do we have events in our graphs that don't have an id?

Reply:
Currently second event in every ledger has this situation, how we add Welcome page shortcut.

Reply:
Ok I see now, it's related to that TODO. I agree we need to handle it. A comment here would be nice for the next time we see it.